src/machine.cpp

```diff
@@ -438,6 +440,16 @@ machine::machine(const machine_config &c, const machine_runtime_config &r) :
     // Register pma board shadow device
     register_pma_entry(make_shadow_pmas_pma_entry(PMA_SHADOW_PMAS_START, PMA_SHADOW_PMAS_LENGTH));
 
+    // TODO(edubart): user should be able to configure these devices
```

Comment:
I think these got disentangled in a future commit I have not yet reached. Just registering the thought here.

Reply:
Indeed, I forgot to remove the line.